Kubernetes/windows/helper.v2.psm1

```diff
@@ -799,6 +915,11 @@ function GetKubeletArguments()
         $kubeletArgs += "--feature-gates=$KubeletFeatureGates"
     }
 
+    if ($IsContainerd) 
+    {
+       $kubeletArgs += @("--container-runtime=remote", "--container-runtime-endpoint=npipe:////./pipe/containerd-containerd")
```

Comment:
Can you keep one arg in a line - Just for readability

Reply:
Can you clarify?